jbang-catalog.json

```diff
@@ -0,0 +1,18 @@
+{
+  "catalogs": {},
+  "aliases": {
+    "pkl": {
+      "script-ref": "org.pkl-lang:pkl-cli-java:0.26.3",
+      "java-agents": []
+    },
+    "pkl-codegen-java": {
+      "script-ref": "org.pkl-lang:pkl-codegen-java:0.26.3",
+      "java-agents": []
+    },
+    "pkl-codegen-kotlin": {
+      "script-ref": "org.pkl-lang:pkl-codegen-kotlin:0.26.3",
```

Comment:
The current version is 0.27.0-SNAPSHOT

Reply:
0.27.0-snapshot is not availabe in maven central so would be bad to point to it.